blog/views.py

```diff
@@ -60,3 +62,21 @@ def edit_post(request, slug):
 
     context = {"form": form}
     return render(request, "blog/add_post.html", context)
+
+
+def register_user(request):
+    form = UserCreationForm()
+    context = {"form": form}
+
+    if request.method == "POST":
+        form = UserCreationForm(request.POST)
+        if form.is_valid():
+            user = form.save(commit=False)
+            user.username = user.username.lower()
```

Comment:
These lines make save lowercase username in a database at the moment of registering a new user.